src/main/java/mods/eln/node/six/SixNodeItem.java

```diff
@@ -31,43 +31,43 @@ public int getMetadata(int damageValue) {
     /**
      * Callback for item usage. If the item does something special on right clicking, he will have one of those. Return True if something happen and false if it don't. This is for ITEMS, not BLOCKS
      */
-    public boolean onItemUse(ItemStack stack, EntityPlayer player, World world, int x, int y, int z, int side, float hitX, float hitY, float hitZ) {
-        Block block = world.getBlock(x, y, z);
-
-        if ((block == Blocks.snow_layer) && ((world.getBlockMetadata(x, y, z) & 0x7) < 1)) {
+    //NOT SURE ABOUT USING getFacingFromVector
```

Comment:
If you're not sure, then don't do it. Also, add a TODO. Anything that needs action later needs a TODO.